{"uuid": "addeac26-459d-4fed-8a13-13af376eb630", "children": ["ec86fb87-d678-4d64-b8d9-52f8f258a1ac", "197bc5eb-aa2b-45ff-a70e-15a2e3a6e68d", "5e3ec810-444c-410d-bc73-be8187a44e2c", "24b8c020-93e0-4dca-b59e-ad1e31ab0441"], "befores": [{"name": "page_context", "status": "broken", "statusDetails": {"message": "NameError: name 'Path' is not defined\n", "trace": "  File \"C:\\Users\\TaimoorKhan\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\TaimoorKhan\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\TaimoorKhan\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\TaimoorKhan\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\TaimoorKhan\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\TaimoorKhan\\Desktop\\Challange_1\\Tests\\test_todo.py\", line 22, in page_context\n    recordings_dir = Path(\"Screen_recordings\")\n                     ^^^^\n"}, "start": 1736422938140, "stop": 1736422938620}], "start": 1736422938140, "stop": 1736422938654}